src/DoubleValueSender.cpp

```diff
@@ -0,0 +1,47 @@
+#include "networktables/NetworkTable.h"
```

Comment:
pro tip - after you're done editing and before you git add, run the cmake target format_all like this: `cmake --build --target format_all`  This will run clang-format on all your .cu, .cpp, and .h files and will format everything consistently

Reply:
Can be done on the build server or otherwise the orin later (not on my Mac).